modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -560,6 +560,92 @@ definition {
 		}
 	}
 
+	@description = "LPS-178784 - Verify that it's possible to send an e-mail notification with Action (Download)"
+	@priority = 5
+	test CanSendEmailViaActionDownload {
+		property portal.acceptance = "true";
+
```

Comment:
Hey @victorhcunha,

When using the MockMock to test, is necessary to add this property: `property test.smtp.server.enabled = "true";`, with this property, the CI sees the email sent.

Reply:
Hi @andreefarias 
Property added!